pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -92,8 +92,8 @@
 
     <RuntimeGroup Include="osx">
       <Parent>unix</Parent>
-      <Architectures>x64</Architectures>
-      <Versions>10.10;10.11;10.12;10.13;10.14;10.15</Versions>
+      <Architectures>x64;arm64</Architectures>
+      <Versions>10.10;10.11;10.12;10.13;10.14;10.15;11;11.0</Versions>
```

Comment:
Why are we adding both `11` and `11.0`?  Typically we only do one or the other because folks would expect them to be equivalent, however its actually directional (11.0 is compatible with 11, but 11 cannot use 11.0 assets).  What value do we calculate?

Reply:
That is what @marek-safar  asked for in  original ticket.
As far as I can tell, dotnet still sees the build as 10.16
```
Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.16
 OS Platform: Darwin
 RID:         osx-x64
 Base Path:   /Users/furt/Downloads/dotnet/sdk/5.0.100-rc.1.20418.3/

```

Reply:
I did more testing and OS report it self as 11.0
```
furt@macos-11 foo % sw_vers
ProductName:	macOS
ProductVersion:	11.0
```
So I removed the `11` to be consistent with the rest. (dotnet still shows 10.16)